components/resources/ipfs_strings.grdp

```diff
@@ -5,6 +5,8 @@
     <message name="IDS_IPFS_ADDRESSES_CONFIG_TITLE" desc="Title of addresses config section">Addresses</message>
     <message name="IDS_IPFS_REPO_STATS_TITLE" desc="Title of repo stats section">Repo Stats</message>
     <message name="IDS_IPFS_NODE_INFO_TITLE" desc="Title of node info section">Node info</message>
+    <message name="IDS_IPFS_NODE_LAUNCH_ERROR" desc="Text of node launch error">Error occured, please try launch agian</message>
+    <message name="IDS_IPFS_NODE_INSTALLATION_ERROR" desc="Text of node installation error">Error occured, please try install node agian</message>
```

Comment:
*agian -> again.

Better yet:
An error occurred, please try to install the node again